js/base/Exchange.js

```diff
@@ -1358,6 +1358,7 @@ module.exports = class Exchange {
                 const [ baseId, quoteId ] = marketId.split (delimiter)
                 const base = this.safeCurrencyCode (baseId)
                 const quote = this.safeCurrencyCode (quoteId)
+                // TODO: What is this? Will this conflict with gateio, okex, and kucoin symbols?
```

Comment:
@kroitor @frosty00 Any comment on this?

Reply:
This is the old code that was used for historical reasons for a few years. We have to rewrite it. Ideally, in a non-breaking way.

Reply:
The approach outlined in this PR probably won't work cause it only does a one-way conversion (on the request compiling side). We either have to do a two-way conversion (on both the request compiling side and the response parsing side), or we have to avoid doing these conversions entirely and just announce a switch to new Binance symbols from a specific version number one or two months ahead.

Reply:
We don't need to do a conversion on the other side, the way this PR is set up is that the new symbols are used, but if someone passes in one of the old symbols, then we just find the new symbol that corresponds to the symbol they passed in

Reply:
Ok, I'll test it out asap, and then we will start preparing the users for the upcoming migration. We will need the time to move the Binance integration to new symbols, that's at least a few weeks, probably 2-3 months.

Reply:
Well it's in an intermittent state right now, I need an answer to this https://github.com/ccxt/ccxt/pull/10720#issuecomment-982254222

Reply:
I reverted the second commit, for the [reasons mentioned here](https://github.com/ccxt/ccxt/pull/10720#issuecomment-983047736)